src/Common/FolderTreeView.cs

```diff
@@ -101,15 +101,21 @@ public string DefaultPath
         private int _scrollviewWidth;
         private int _scrollviewHeight;
         private readonly Dictionary<string, int> _itemHeightMap = new Dictionary<string, int>();
+        private readonly HashSet<string> _displayItems = new HashSet<string>();
 
         public void DrawDirectoryTree()
         {
             ExpandToCurrentFolder();
 
-            if (_selectedTreeScrollPositionY.HasValue && Event.current.type == EventType.Layout)
+            if (Event.current.type == EventType.Layout)
             {
-                _treeScrollPosition.y = Mathf.Max(0, _selectedTreeScrollPositionY.Value - _scrollviewHeight * 0.5f);
-                _selectedTreeScrollPositionY = null;
+                _displayItems.Clear();
```

Comment:
```suggestion
                // Use _displayItems instead of calculating every time to fix a bug that sometimes causes exception when scrolling
                // Scrolling may cause the item to be shown or hidden in the middle of an event, changing the layout function call
                _displayItems.Clear();
```